Game/Messages/Server/S_GUILD_QUEST_LIST.cs

```diff
@@ -70,6 +70,10 @@ public GuildQuest ActiveQuest()
             return GuildQuests.Where(x => x.Active == true).FirstOrDefault();
         }
 
+        public List<GuildQuest> ActiveQuests()
+        {
+            return GuildQuests.Where(x => x.Active == true).ToList();
```

Comment:
FYI, you do not need to check `bool == true`.  Omitting the `== true` still keeps the logic intact.